.github/workflows/docker.yml

```diff
@@ -0,0 +1,87 @@
+# NOTE: keep this action in sync with `docker-tags.yml`.
+
+name: Publish Docker
+on:
+  pull_request:
```

Comment:
Let's check whether we can make this not required or maybe even trigger-only for PRs to not add too much time before we get a ✅ from CI.